sdk/lib/crt/msvcrtex.cmake

```diff
@@ -41,7 +41,10 @@ list(APPEND MSVCRTEX_SOURCE
     misc/iswblank.c
     misc/ofmt_stub.c)
 
-if(NOT MSVC)
+if(MSVC)
+    list(APPEND MSVCRTEX_SOURCE
+        startup/threadSafeInit.c)
```

Comment:
What did we use for MSVC before ?!

Reply:
It is disabled on MVC builds, but Clang-CL doesn't seem to care.